scalableminds/webknossos
# Scale to resolution, data loading for anisotropic datasets

### Mailable description of changes:
Backend supports importing + loading of anisotropic datasets. (Fontend changes to come)

### Scale vs Resolution

Unify per-layer scale and resolution. Now `scale` exists only once per dataset (again) and each layer has a list of `resolution`s, each of which can either be a scalar or a triple.

### Steps to test:
- import a dataset that didn’t have a json
- “edit” a dataset that already had a json
- in both cases there should now be no `scale` fields in the layers (instead `resolution` fields which are scalars for the old datasets (powers of two), and arrays-of-three for anisotropic datasets
- data loading should still work for old datasets

### Issues:
- contributes to #2295

------
